charts/seed-monitoring/charts/core/charts/prometheus/templates/config.yaml

```diff
@@ -627,6 +627,28 @@ data:
         replacement: 'shoot_node_logging_incoming_$1'
 {{- end }}
 
+{{- if .Values.reversedVPN.enabled }}
+    - job_name: reversed-vpn-envoy-side-car
+      honor_labels: false
```

Comment:
https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config default is false
```suggestion
```